src/routes/safe/components/Transactions/helpers/EditableTxParameters.tsx

```diff
@@ -33,10 +29,8 @@ export const EditableTxParameters = ({
 }: Props): React.ReactElement => {
   const [isEditMode, toggleEditMode] = useState(false)
   const [useManualValues, setUseManualValues] = useState(false)
-  const threshold = useSelector(currentSafeThreshold) || 1
-  const defaultParameterStatus = isOffChainSignature && threshold > 1 ? 'ETH_HIDDEN' : 'ENABLED'
```

Comment:
Why can the default be removed?

Reply:
I removed the whole logic that was showing/hiding eth params based on this. We now completely hide or show the entire form depending on whether it's execution or not.